.github/workflows/backend-Publish.yaml

```diff
@@ -0,0 +1,48 @@
+name: Publish backend Docker image
+
+on:
+  push:
+    branches:
+      - 'main'
+
+jobs:
+
+  paths-filter:
+    runs-on: ubuntu-22.04
+
+    outputs:
```

Comment:
I was thinking that since this is a mono-repo for different projects, we would want specific builds to happen only when files in the particular project changed.

Reply:
Yes, and I've asked you to use Github Actions's own feature for this. You don't need those hacks to do it.

Reply:
oh, okay. I am fairly new to CI/CD and I didn't know about it. I have seen it now and I will update it. Thanks.